foursight_core/react/api/react_api.py

```diff
@@ -451,6 +452,9 @@ def _create_user_record_from_input(self, user: dict, include_deletes: bool = Fal
                 del user["role"]
             if "roles" in user:
                 del user["roles"]
+            if "status" in user:
+                if not user["status"] or user["status"] == "-":
```

Comment:
Is this `"-"` convention something standard or is it our own oddity?

Reply:
That's our (my) own oddity; should really handle in UI, was in hurry, will put a comment about it.